config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -970,7 +970,9 @@ public ServerHttpSecurity oauth2Login(Customizer<OAuth2LoginSpec> oauth2LoginCus
 
 		private ServerOAuth2AuthorizedClientRepository authorizedClientRepository;
 
-		private ReactiveAuthenticationManager authenticationManager;
+                private ReactiveAuthenticationManager authenticationManager;
```

Comment:
Something odd happened to the whitespace here, could you correct that please?

Reply:
Done, sorry for the whitespace noice